tests/utils.py

```diff
@@ -201,10 +228,25 @@ def assert_code_equal(actual, expected):
 get_classification_binary_random_data_model_trainer = functools.partial(
     ModelTrainer.get_instance, "classification_binary_rnd")
 
+
 get_bounded_regression_model_trainer = functools.partial(
     ModelTrainer.get_instance, "boston_y_bounded")
 
 
+get_regression_with_missing_values_model_trainer = functools.partial(
```

Comment:
For the consistency with all other names from this PR.

```suggestion
get_regression_w_missing_values_model_trainer = functools.partial(
```